compiler/semcall.nim

```diff
@@ -416,6 +416,8 @@ proc resolveOverloads(c: PContext, n, orig: PNode,
             let msg = errUndeclaredField % field & " for type " & getProcHeader(c.config, sym)
             localError(c.config, orig[2].info, msg)
         else:
+          if n.kind == nkCall and n[0].kind == nkDotExpr:
+            return result
```

Comment:
It's not easy to see how skipping the error report always ensures that an other, better error is produced.